packages/stargate/src/multisignature.spec.ts

```diff
@@ -0,0 +1,254 @@
+        extraBitsStored: 2,
+      });
+      expect(
+        makeCompactBitArray([false, false, false, false, false, false, false, false, false, true]),
+      ).toEqual({
+        elems: new Uint8Array([0b00000000, 0b01000000]),
+        extraBitsStored: 2,
+      });
+    });
+  });
+
+  describe("makeMultisignedTx", () => {
+    it("works", async () => {
+      pendingWithoutSimapp();
+      const wallet0 = await Secp256k1HdWallet.fromMnemonic(faucet.mnemonic, makeCosmoshubPath(0));
+      const wallet1 = await Secp256k1HdWallet.fromMnemonic(faucet.mnemonic, makeCosmoshubPath(1));
+      const wallet2 = await Secp256k1HdWallet.fromMnemonic(faucet.mnemonic, makeCosmoshubPath(2));
+      const wallet3 = await Secp256k1HdWallet.fromMnemonic(faucet.mnemonic, makeCosmoshubPath(3));
+      const wallet4 = await Secp256k1HdWallet.fromMnemonic(faucet.mnemonic, makeCosmoshubPath(4));
+      const pubkey0 = encodeSecp256k1Pubkey((await wallet0.getAccounts())[0].pubkey);
+      const pubkey1 = encodeSecp256k1Pubkey((await wallet1.getAccounts())[0].pubkey);
+      const pubkey2 = encodeSecp256k1Pubkey((await wallet2.getAccounts())[0].pubkey);
+      const pubkey3 = encodeSecp256k1Pubkey((await wallet3.getAccounts())[0].pubkey);
+      const pubkey4 = encodeSecp256k1Pubkey((await wallet4.getAccounts())[0].pubkey);
+      const address0 = (await wallet0.getAccounts())[0].address;
+      const address1 = (await wallet1.getAccounts())[0].address;
+      const address2 = (await wallet2.getAccounts())[0].address;
+      const address3 = (await wallet3.getAccounts())[0].address;
+      const address4 = (await wallet4.getAccounts())[0].address;
+      const multisigPubkey = createMultisigThresholdPubkey([pubkey0, pubkey1, pubkey2, pubkey3, pubkey4], 2);
+      const multisigAddress = pubkeyToAddress(multisigPubkey, "cosmos");
+      expect(multisigAddress).toEqual("cosmos1h90ml36rcu7yegwduzgzderj2jmq49hcpfclw9");
+
+      const client0 = await SigningStargateClient.connectWithSigner(simapp.tendermintUrl, wallet0);
+      const client1 = await SigningStargateClient.connectWithSigner(simapp.tendermintUrl, wallet1);
+      const client2 = await SigningStargateClient.connectWithSigner(simapp.tendermintUrl, wallet2);
+      const client3 = await SigningStargateClient.connectWithSigner(simapp.tendermintUrl, wallet3);
+      const client4 = await SigningStargateClient.connectWithSigner(simapp.tendermintUrl, wallet4);
+
+      const msgSend: MsgSend = {
+        fromAddress: multisigAddress,
+        toAddress: "cosmos19rvl6ja9h0erq9dc2xxfdzypc739ej8k5esnhg",
+        amount: coins(1234, "ucosm"),
+      };
+      const msg = {
+        typeUrl: "/cosmos.bank.v1beta1.MsgSend",
+        value: msgSend,
+      };
+      const gasLimit = 200000;
+      const fee = {
+        amount: coins(2000, "ucosm"),
+        gas: gasLimit.toString(),
+      };
+      const memo = "Use your tokens wisely";
+
+      const multisigAccount = await client0.getAccount(multisigAddress);
```

Comment:
How does this work?

Reply:
`multisigAddress` is a regular address that exists on chain (via the new genesis entries I created). This is a rea-only call that does not require's `wallet0`.

However, I am not sure if that answers the questin.

Reply:
Ah yes, it does, I was thinking this was getting the account from the wallet.